gcp/modules/templater/values/couchdb.yaml

```diff
@@ -1,6 +1,12 @@
-# clusterSize is the initial size of the CouchDB cluster.
-clusterSize: 2
+clusterSize: ${env == "dev" ? 2 : 3}
 
 adminUsername: ${couchdb_admin_username}
 adminPassword: ${couchdb_admin_password}
 cookieAuthSecret: ${couchdb_secret}
+
+persistentVolume:
```

Comment:
I don't understand what this stanza does. Can you tell me more?

Reply:
Check https://github.com/gpii-ops/gpii-infra/blob/master/common/charts/couchdb/values.yaml#L28

Reply:
Ok